readme.md

```diff
@@ -281,6 +281,7 @@
 - [Speech and Natural Language Processing](https://github.com/edobashira/speech-language-processing#readme)
 	- [Spanish](https://github.com/dav009/awesome-spanish-nlp#readme)
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby#readme)
+	- [Question Answering](https://github.com/seriousmac/awesome-qa#readme)
```

Comment:
Also please provide a good description for the list:

> - Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

Reply:
All about asking in natural language and answering with a machine